PowerEditor/src/WinControls/Preference/preference_rc.h

```diff
@@ -123,16 +123,17 @@
 	#define	IDC_BORDERWIDTH_SLIDER (IDD_PREFERENCE_SUB_EDITING + 33)
 	#define	IDC_CHECK_DISABLEADVANCEDSCROLL (IDD_PREFERENCE_SUB_EDITING + 34)
 	#define	IDC_CHECK_NOEDGE (IDD_PREFERENCE_SUB_EDITING + 35)
-	#define	IDC_CHECK_SCROLLBEYONDLASTLINE (IDD_PREFERENCE_SUB_EDITING + 36)
-
-	#define	IDC_STATIC_MULTILNMODE_TIP (IDD_PREFERENCE_SUB_EDITING + 37)
-	#define	IDC_COLUMNPOS_EDIT (IDD_PREFERENCE_SUB_EDITING + 38)
-	#define	IDC_CHECK_RIGHTCLICKKEEPSSELECTION (IDD_PREFERENCE_SUB_EDITING + 39)
-	#define	IDC_PADDINGLEFT_SLIDER (IDD_PREFERENCE_SUB_EDITING + 40)
-	#define	IDC_PADDINGRIGHT_SLIDER (IDD_PREFERENCE_SUB_EDITING + 41)
-	#define	IDC_PADDINGLEFTVAL_STATIC (IDD_PREFERENCE_SUB_EDITING + 42)
-	#define	IDC_PADDINGRIGHTVAL_STATIC (IDD_PREFERENCE_SUB_EDITING + 43)
-	#define	IDC_DISTRACTIONFREEVAL_STATIC (IDD_PREFERENCE_SUB_EDITING + 44)
+	#define	IDC_CHECK_VIRTUALSPACE (IDD_PREFERENCE_SUB_EDITING + 36)
+	#define	IDC_CHECK_SCROLLBEYONDLASTLINE (IDD_PREFERENCE_SUB_EDITING + 37)
+
+	#define	IDC_STATIC_MULTILNMODE_TIP (IDD_PREFERENCE_SUB_EDITING + 38)
+	#define	IDC_COLUMNPOS_EDIT (IDD_PREFERENCE_SUB_EDITING + 39)
+	#define	IDC_CHECK_RIGHTCLICKKEEPSSELECTION (IDD_PREFERENCE_SUB_EDITING + 40)
+	#define	IDC_PADDINGLEFT_SLIDER (IDD_PREFERENCE_SUB_EDITING + 41)
+	#define	IDC_PADDINGRIGHT_SLIDER (IDD_PREFERENCE_SUB_EDITING + 42)
+	#define	IDC_PADDINGLEFTVAL_STATIC (IDD_PREFERENCE_SUB_EDITING + 43)
+	#define	IDC_PADDINGRIGHTVAL_STATIC (IDD_PREFERENCE_SUB_EDITING + 44)
+	#define	IDC_DISTRACTIONFREEVAL_STATIC (IDD_PREFERENCE_SUB_EDITING + 45)
 

```

Comment:
Please revert the changed IDs as it breaks translation. IDC_CHECK_VIRTUALSPACE should then be:
```
#define	IDC_CHECK_VIRTUALSPACE (IDD_PREFERENCE_SUB_EDITING + 45)
```

Reply:
Done